src/modules/Rating/RatingIcon.js

```diff
@@ -52,7 +70,16 @@ export default class RatingIcon extends Component {
       useKeyOnly(selected, 'selected'),
       'icon'
     )
+    const rest = _.omit(this.props, _.keys(RatingIcon.propTypes))
 
-    return <i className={classes} onClick={this.handleClick} onMouseEnter={this.handleMouseEnter} />
+    return (
+      <i role='radio' tabIndex={0}
+        {...rest}
+        className={classes}
+        onKeyPress={this.handleKeyPress}
```

Comment:
Should this be `onKeyUp`?  Key press is fired on key down and repeatedly fires when held.  Whereas, the `keyup` event only fires once on release.  I believe `onKeyUp` is the correct event handler for adding button like attributes to non-button elements, since it mirrors the mouse click behavior, only invoked once and invoked on release.